src/main/java/it/gov/pagopa/payhub/activities/service/ingestionflow/organization/OrganizationProcessingService.java

```diff
@@ -0,0 +1,93 @@
+import it.gov.pagopa.payhub.activities.connector.organization.BrokerService;
+import it.gov.pagopa.payhub.activities.connector.organization.OrganizationService;
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationErrorDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationIngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationIngestionFlowFileResult;
+import it.gov.pagopa.payhub.activities.mapper.ingestionflow.organization.OrganizationMapper;
+import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowProcessingService;
+import it.gov.pagopa.pu.organization.dto.generated.Broker;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import it.gov.pagopa.pu.processexecutions.dto.generated.IngestionFlowFile;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Service
+@Lazy
+@Slf4j
+public class OrganizationProcessingService extends IngestionFlowProcessingService<OrganizationIngestionFlowFileDTO, OrganizationIngestionFlowFileResult, OrganizationErrorDTO> {
+
+    private final OrganizationMapper organizationMapper;
+    private final OrganizationService organizationService;
+    private final BrokerService brokerService;
+
+
+    public OrganizationProcessingService(
+            OrganizationMapper organizationMapper,
+            OrganizationErrorsArchiverService organizationErrorsArchiverService,
+        OrganizationService organizationService, BrokerService brokerService) {
+        super(organizationErrorsArchiverService);
+        this.organizationMapper = organizationMapper;
+        this.organizationService = organizationService;
+        this.brokerService = brokerService;
+    }
+
+
+    public OrganizationIngestionFlowFileResult processOrganization(
+            Iterator<OrganizationIngestionFlowFileDTO> iterator,
+            List<CsvException> readerException,
+            IngestionFlowFile ingestionFlowFile, Path workingDirectory) {
+        List<OrganizationErrorDTO> errorList = new ArrayList<>();
+        OrganizationIngestionFlowFileResult ingestionFlowFileResult = new OrganizationIngestionFlowFileResult();
+        ingestionFlowFileResult.setOrganizationIpaCodeList(new ArrayList<>());
+
+        process(iterator, readerException, ingestionFlowFileResult, ingestionFlowFile, errorList, workingDirectory);
+        return ingestionFlowFileResult;
+    }
+
+    @Override
+    protected boolean consumeRow(long lineNumber, OrganizationIngestionFlowFileDTO organizationDTO, OrganizationIngestionFlowFileResult ingestionFlowFileResult, List<OrganizationErrorDTO> errorList, IngestionFlowFile ingestionFlowFile) {
+        try {
+            Broker broker = brokerService.getBrokerByFiscalCode(organizationDTO.getBrokerCf());
```

Comment:
should all Brokers be the same and its fiscalCode be equal to the organization fiscalCode related to the input ingestionFlowFile?
This way, we allow to accept organizations by broker, relating the input file to the broker itself